sky/engine/core/script/dart_init.cc

```diff
@@ -387,9 +395,11 @@ void InitDartVM() {
     // The version of the VM setup to run precompiled code does not recognize
     // the mirrors or the background compilation flags. They are never enabled.
     // Make sure we dont pass in unrecognized flags.
-    args.append(kDartMirrorsArgs, arraysize(kDartMirrorsArgs));
-    args.append(kDartBackgroundCompilationArgs,
-                arraysize(kDartBackgroundCompilationArgs));
+    if (IsDevelopMode()) {
+      args.append(kDartMirrorsArgs, arraysize(kDartMirrorsArgs));
```

Comment:
Why do we remove these in non-develop mode?  We never allow mirrors and always want background comp, or?